### DISCUSSION TITLE
v0.7.0 Feature Freeze April 6th

### DISCUSSION THREAD

**Discussion by shaneutt** [kubernetes-sigs/gateway-api#1892] [Announcements]
Hi everyone :wave: 

We (the maintainers) are planning to freeze the [v0.7.0 milestone](https://github.com/kubernetes-sigs/gateway-api/milestone/7) on April 6th, 2023. This means that any issues or PRs which are not marked as `release-blocker` which are not closed/merged by the end of that day will overflow to the following minor release [v0.8.0](https://github.com/kubernetes-sigs/gateway-api/milestone/13). At that point we intend to produce an RC-0 release from `main` so we can submit that for upstream SIG Network review and get the process started towards releasing `v0.7.0`.

If you have any questions or concerns please reach out here in the repository, or in `#sig-network-gateway-api` on [Kubernetes Slack](https://kubernetes.slack.com)!

**Comment by zirain** [comment]
@shaneutt when will v0.7.0-rc1 release?

**Reply by shaneutt** [reply]
The [description of the v0.7.0 milestone](https://github.com/kubernetes-sigs/gateway-api/milestone/7) includes some rough ideas of RC and release timing, from that description you'll see that we're (again, _roughly_) estimating April 13th for RC1 specifically. Let us know if you have any other questions or concerns!

**Reply by zirain** [reply]
@shaneutt will v0.7.0 release before KubeCon EU?

**Reply by shaneutt** [reply]
No, sorry. But it's possible we'll get the first release candidate out _during_ Kubecon.
